examples/gno.land/p/demo/blog/blog.gno

```diff
@@ -55,29 +46,30 @@ func (b Blog) RenderHome(res *mux.ResponseWriter, req *mux.Request) {
 func (b Blog) RenderPost(res *mux.ResponseWriter, req *mux.Request) {
 	slug := req.GetVar("slug")
 
-	post, found := b.Posts.Get(slug)
-	if !found {
+	post := b.Posts.GetPost(slug)
```

Comment:
as here you're using `GetPost` on `Posts` anyway, not `b`. So this would become `b.Posts.BySlug(slug)`